.github/workflows/license-compliance-go.yml

```
@@ -0,0 +1,65 @@
+      continue-on-error: true
+
+    # Check for forbidden licenses
+    - name: Check for forbidden licenses
+      working-directory: ./cla-backend-legacy
+      run: |
+        echo "Checking for GPL, LGPL, AGPL licenses..."
+        go-licenses check ./... | grep -i -E "(gpl|agpl)" && exit 1 || echo "No forbidden licenses found"
+      continue-on-error: true
+
+    - name: Upload license report
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.11)

[error] the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)